00-HTML-CSS-basics/html/form.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<title>Sign up</title>
+		<meta name="viewport" content="width=device-width,initial-scale=1"/>
+		<meta name="author" content="Braulio Evangelisti"/>
+		<meta name="description" content="Sign up form example">
+	</head>
+	<body>
+		<header>
+			<h1>Sign up form!</h1>
+		</header>
+		<section>
+			<h2>Fill the form to sign up:</h2>
+			<form action="signin.php" method="POST">
+				<fieldset>
+					<legend>Sign up details</legend>
+					<div>
+						<label for="fname">First Name:</label><br>
```

Comment:
**Minor:** just as a thought, we try to avoid using ``<br />`` tags, since we're going to handle it with CSS (it's not incorrect though, just something to have in mind).